fixtures/codegen/todolist.simple-fixed.yml

```diff
@@ -0,0 +1,187 @@
+swagger: '2.0'
```

Comment:
what does the `-fixed` mean? What was fixed?

Reply:
You have been using invalid spec for codegen unit testing for quite a while.
It doesn't hurt since you are using low level templating UT, but it's not workable for full code generation.

"-fixed" means that this is a valid spec.

I tried to change as little as possible because of the code pattern matching in UT.
Hence, in cases where I wished to non-regress your fixtures, I add to make a copy and make it a valid spec.

Changes: 
- todolist.simple-fixed.yml : default value must validate the data type, path params are required
- azure-analysis: same as ms-cog i canary (duplicate property in allOf)

I introduced reuse codegen specs in a new CI test run (./hack/codegen-nonreg.sh) to extend our testing capabilities. However many fixtures fail for full generation. All this is explained as comments in the test script.